staging/src/k8s.io/client-go/rest/request_test.go

```diff
@@ -2218,3 +2219,41 @@ func TestThrottledLogger(t *testing.T) {
 		t.Fatalf("expected %v log messages, but got %v", e, a)
 	}
 }
+
+func TestRequestMaxRetries(t *testing.T) {
+	successAfterRetry := 5
+	retried := 0
```

Comment:
extremely pedantic nit: the first call is not a retry. Maybe `calls` or `tries` is a better name.

(actually it looks like maybe you were counting retry headers sent? I think it's better to count requests sent by the client, then the test can make better guarantees about the behavior.)

Reply:
renamed the variables to `successAtNthCall` and `calls`